Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
> To understand you. In my Extension class

Yes! That is 100% correct :grin: 

> And then the same for ExampleController class with different filenames to include this files only on the routes served by the controller?

This is also correct.

Basically it means that those functions are only run during the request cycle.